src/components/AssetDescriptionComponent.jsx

```diff
@@ -24,82 +26,66 @@ const AssetDescriptionComponent = props => (
         </div>
       </Grid.Column>
       <Grid.Column>
-        <Confirm
-          content="Are you sure you want to make this change?"
-          open={props.open}
-          cancelButton={
-            <ButtonComponent
-              className="cancel"
-              buttonName="Cancel"
-              handleClick={props.handleCancel}
-            />
-          }
-          confirmButton={
-            <ButtonComponent
-              className="save"
-              buttonName="Save"
-              color="primary"
-              buttonState={props.buttonState}
-              handleClick={props.handleConfirm}
-            />
-          }
-        />
-        {(!isEmpty(values(props.assignedUser))) ?
-          <div id="allocate-asset">
-            <Header as="h3" content="Assigned To:" />
-            <div
-              id="email"
-              className="asset-specs"
-            >
-              {props.assignedUser.email}
+        {
+          (!isEmpty(values(props.assignedUser)))
```

Comment:
The ternary operator is not the best option here as it reduces the code comprehension.  Create another component for the assigned to and use that here. E.g.
```
const AssignedTo = ({ assignedUser }) => {
  if (isEmpty(assignedUser)) {
    return (
      <div id="allocate-asset">
        <Header as="h3" content="Assign this asset to:" />
        <DropdownComponent
          customClass="form-dropdown"
          label="Assign this asset to:"
          placeHolder="Select Andela Email To Assign This Asset"
          name="assign-user"
          search
          value={props.selectedUserId}
          onChange={props.onSelectUserEmail}
          options={userEmailsOptions(props.users)}
         />
      </div>
    );
  }

  return (
    <div id="allocate-asset">
      <Header as="h3" content="Assigned To:" />
      <div
        id="email"
        className="asset-specs"
       >
         {props.assignedUser.email}
       </div>
    </div>
  );
};
```
Then use the `AssignedTo` here, passing it `assignedUser` as prop